### PR TITLE
feat: c4c improvement

### PR DESCRIPTION
**Pull request by yuesth**
1. curating improvement
  - [ x ] label card on card category submission
  - [ x ] select and reject all button
  - [ x ] sort by 

2. owned card indicator
  - [ x ] icon owned on card and card detail
  - [ x ] warning on payment confirmation
  - [ x ] error message for submit new card
  - [ x ] notif for diqualified user

3. submit existing card to category
  - [ x ] user flow for submit existing card

### CODE REVIEW

**Comment by ahnafalfariza** [comment]
image
kalo dia category lain modalnya c4c gan.

**Review by emarai — DISMISSED** [review]

File: components/TokenSeries/CardList.js [hunk 1]
```diff
@@ -146,6 +148,21 @@ const TokenSeriesSingle = ({
...
+			  }
+			: null,
+		(key) => {
+			return ParasRequest.get(`${process.env.V2_API_URL}/token`, {
+				params: key,
+			})
+		}
+	)
+
 	const store = useStore()
```
Comment (ahnafalfariza) [inline components/TokenSeries/CardList.js:166]: ini emang supposed to fetch /token lagi di CardList? soalnya kalo gua liat network jadi bakal banyak fetch single token dari setiap item di /token-series. 

jadi misal fetch `/token-series` => Array(10), bakal fetch `/token` 10x

any suggestion ga? atau mungkin bisa di kasih conditional aja kalo dia category c4c aja fetchnya

File: components/TokenSeries/TokenSeriesDetail.js [hunk 2]
```diff
@@ -158,6 +160,21 @@ const TokenSeriesDetail = ({ token, className, isAuctionEnds }) => {
...
+					owner_id: currentUser,
+			  }
+			: null,
+		(key) => {
+			return ParasRequest.get(`${process.env.V2_API_URL}/token`, {
+				params: key,
+			})
+		}
+	)
+
```
Comment (ahnafalfariza) [inline components/TokenSeries/TokenSeriesDetail.js:177]: casenya sama kayak diatas

**Review by ahnafalfariza — COMMENTED** [review]
lebih ke improvement fetchingnya aja gan, actionnya mau gimana bisa di discuss lebih lanjut

File: components/TokenSeries/CardList.js (hunk 1 above)
Comment (yuesth) [reply components/TokenSeries/CardList.js:166]: bisa sih hu dikasih conditional fetch cuma di category nya c4c

File: components/Modal/AddExistCardCategoryModal.js [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+								>
+									<div
+										className="p-3 hover:bg-gray-500 cursor-pointer flex items-center whitespace-nowrap"
+										onClick={() => {
+											setTokens(
+												AllTokens.filter(
+													(token) => !selectedTokens.map((data) => data._id).includes(token._id)
+												)
+											)
+											setFilterCollection('All Collections')
```
[the remark this answers is not in the document]
Comment (yuesth) [reply components/Modal/AddExistCardCategoryModal.js:173]: woke huu

File: components/Modal/AddExistCardCategoryModal.js [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+											<p className="mt-4 text-center underline cursor-pointer">Submit New Card?</p>
+										</Link>
+									</>
+								)}
+							</div>
+						</div>
+					) : (
+						<div
+							id="existingCardCategory"
+							className="mt-2 grid grid-cols-3 md:grid-cols-5 gap-2 w-full max-h-[230px] overflow-y-auto"
```
[the remark this answers is not in the document]
Comment (yuesth) [reply components/Modal/AddExistCardCategoryModal.js:250]: woke thankyou hu

**Review by iqbalutomo — APPROVED** [review]

**Review by ahnafalfariza — APPROVED** [review]

**Pull request merged**